Doc/library/time.rst

```diff
@@ -21,10 +21,9 @@ An explanation of some terminology and conventions is in order.
 
 .. index:: single: epoch
 
-* The :dfn:`epoch` is the point where the time starts, and is platform
-  dependent.  For Unix and Windows, the epoch is January 1, 1970, 00:00:00 (UTC).
-  To find out what the epoch is on a given platform, look at
-  ``time.gmtime(0)``.
+* The :dfn:`epoch` is the point where the time starts.
+  It is January 1, 1970, 00:00:00 (UTC) on all platforms.
+  To find out what the epoch is, look at ``time.gmtime(0)``.
```

Comment:
This reads a bit strange now - you tell us exactly what the epoch is, and then tell us how to find out what it is. How about changing the above 3 lines to something like:

* The :dfn:`epoch` is the point where the time starts, the
  return value of ``time.gmtime(0)``.
  It is January 1, 1970, 00:00:00 (UTC) on all platforms.

Reply:
I agree that the doc sounds strange. But I wanted to keep "gmtime(0)", it sounds important to me. I like your wording, I copied it in my PR ;-) Would you mind to review it again?